fedimint-cli/src/lib.rs

```diff
@@ -761,19 +794,37 @@ impl FedimintCli {
                     .map_err_cli_general()?;
                 Ok(CliOutput::Raw(serde_json::Value::Null))
             }
-            Command::Dev(DevCmd::DecodeInviteCode { invite_code }) => {
-                Ok(CliOutput::DecodeInviteCode {
+            Command::Dev(DevCmd::Decode { decode_type }) => match decode_type {
+                DecodeType::InviteCode { invite_code } => Ok(CliOutput::DecodeInviteCode {
                     url: invite_code.url(),
                     federation_id: invite_code.federation_id(),
-                })
-            }
-            Command::Dev(DevCmd::EncodeInviteCode {
-                url,
-                federation_id,
-                peer,
-            }) => Ok(CliOutput::InviteCode {
-                invite_code: InviteCode::new(url, peer, federation_id),
-            }),
+                }),
+                DecodeType::Notes { notes } => Ok(CliOutput::DecodeNotes {
+                    federation_id_prefix: notes.federation_id_prefix().to_string(),
+                    notes: notes.notes().clone(),
```

Comment:
I actually liked the previous version of your JSON encoding code which was able to display all `OOBNoteData` variants. Now we are missing invite codes when provided.

Reply:
updated to use it on OOB notes through .notes_json